<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screen Clicker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="inp_image_path">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_load_image">
        <property name="text">
         <string>Scegli Immagine</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_scan">
        <property name="text">
         <string>Scansiona</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_stop">
        <property name="text">
         <string>Ferma</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Crop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbx_x">
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbx_y">
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbx_width">
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>1920</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbx_height">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>1080</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lbl_image">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
